### ISSUE TITLE
Missing line when generating for `Metals` (Scala)

### ISSUE THREAD

**Issue by leana8959** [wintermute-cell/gitignore.nvim#4]
Hello,

I noticed that, when I was trying to generate `.gitignore` for `Metals`, the `.metals/` line is missing.  
I checked the template file in this repo, it was present there, am I somehow misconfiguring this plugin?

Desired output: 
```gitignore
.metals/
.bloop/
project/**/metals.sbt
```
Actual output:
```gitignore
.bloop/
project/**/metals.sbt
```

Thank you for your time, have a nice day :)

**Comment by wintermute-cell** [comment]
Hey leana,

thank you for the bug report! It was an issue in the `collapseEmptyStrings` function (now called `collapseEmptyLines`). Fixed with b61e4a50b78f9a3702a4ed168ab6c9ceba5f6299.

Have a nice day too :)

**Comment by leana8959** [comment]
It is indeed working now, awesome!  
Thank you for the quick fix!
